framework/base/Arrayable.php

```diff
@@ -82,11 +82,11 @@ public function extraFields();
      *
      * @param array $fields the fields that the output array should contain. Fields not specified
      * in [[fields()]] will be ignored. If this parameter is empty, all fields as specified in [[fields()]] will be returned.
-     * @param array $expand the additional fields that the output array should contain.
+     * @param array $except the additional fields that the output array should contain.
```

Comment:
I don't think this name change is needed. These are still list of fields, not exceptions.